stack.yaml

```diff
@@ -17,7 +17,7 @@
 #
 # resolver: ./custom-snapshot.yaml
 # resolver: https://example.com/snapshots/2018-01-01.yaml
-resolver: lts-12.24
+resolver: lts-18.19
```

Comment:
Did you need to change the resolver version?  Wha is the *oldest* LTS we want support?  Perhaps still the LTS for GHC 8.0?  That would actually be 9.21!   WIth 12.24 being GHC 8.4.4, I think we should let it stand at that or older for now???

Reply:
Reverted change - originally I was receiving a build error but after cleaning `~/.stack` and `stack clean --full` and reverting version I no longer receive the error.

Original error was along the lines of hugs is no longer supported.

Reply:
I don't have any opinions about the LTS version.